tests/samplers_tests/test_samplers.py

```diff
@@ -549,3 +550,27 @@ def after_trial(
     study.tell(study.ask(), 1.0)
 
     assert n_calls == 1
+
+
+@parametrize_sampler
+def test_sample_single_distribution(sampler_class: Callable[[], BaseSampler]) -> None:
+
+    relative_search_space = {
+        "a": UniformDistribution(low=1.0, high=1.0),
+        "b": LogUniformDistribution(low=1.0, high=1.0),
+        "c": DiscreteUniformDistribution(low=1.0, high=1.0, q=1.0),
+        "d": IntUniformDistribution(low=1, high=1),
+        "e": IntLogUniformDistribution(low=1, high=1),
+        "f": CategoricalDistribution([1]),
+    }
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", optuna.exceptions.ExperimentalWarning)
+        sampler = sampler_class()
+    study = optuna.study.create_study(sampler=sampler)
+
+    for _ in range(10):
```

Comment:
Is not a single trial enough, given that samplers are configured in such a way (e.g. with `n_startup_trials=0`)?

Reply:
We need to test the construction of the model, so I think we should set `n_trials >= 2`. I changed it to 2 from 10. Thanks for the comment.

Reply:
Could you leave a comment in the code about that?